mean_shift_sklearn.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+import argparse
+import matplotlib.pyplot as plt
+from itertools import cycle
+import numpy as np
+from sklearn.cluster import MeanShift
+
+information = """
+Calculates the mean_shift/peaks for the given dataset.
+
+Input:
+* bed6 file containing dataset
+* bandwidth (an integer value) to calculate the cluster radius/range
+* Output file name (optional)
+* Output_Graph file name (optional)
+
+Output:
+* bed6 file with new_centroids/peaks for the given data set.
+ default name "output.bed".
+* graph file, default name "graph.png".
+
+Example:
+- read file input.bed and write new peaks/centroids into output.bed
+- mean_shift.py input.bed 200 -o data_output.bed -g output-graph.png
+
+"""
+
+
+class Mean_Shift():
+
+    """
+    calcualtes the mean shift
```

Comment:
better: applies mean shift algorithm to an array of integers

Reply:
done